db/1-schema.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE IF NOT EXISTS ACCOUNTS (
```

Comment:
The intent first pass was to get sql artifacts loaded into postgres for dev purposes. Currently, you have to perform a `docker-compose rm db` and re-run to get the .sql files loaded. However, we'll definitely need a more formal process to handle structural db updates.